src/Docs/Resources/current/4-how-to/670-administration-language-switcher.md

```diff
@@ -0,0 +1,256 @@
+[titleEn]: <>(Switching languages in administration and storefront)
+[metaDescriptionEn]: <>(This HowTo will teach you how the editing language is changed and what impacts these changes have.)
+
+## Overview
+
+The editing language of the administration is one of the key USPs of its user interface.
+It is present all over the application.
+The most important fact is that it is used change for everything at once to keep editing consistent.
+For example if you change the editing language in the language dropdown every listing or detail page will update and use that new language to display and edit content.
+To make sure your custom entity supports translations look at the [overview of translations in the DAL](../../2-internals/1-core/20-data-abstraction-layer/120-translations.md). 
+
+
+## Change language in a listing
+
+To change the language in an administration listing you have to add the language switching component into your action bar:
+
+```twig
+{# foobar-list.html.twig #}
+
+<sw-page>
+    <template slot="language-switch">
+        <sw-language-switch></sw-language-switch>
+    </template>
+</sw-page>
+```
+
+Although the slot already has the name it does not serve the language switch itself.
+The slot is used to place the language switcher consistently on all pages at the same place.
+But why is it not served there by default.
+At first not every component is translatable and should not look like it supports it.
+And the second reason is that the page does not know what to do when the language switches.
+The language switcher just tells the administration application to change the language and every request afterwards uses the new language.
+So as our content is not automatically reloaded on switching the language you have to listen to the change event of the language switcher like this:
+
+```twig
+<sw-language-switch v-on:on-change="getList"></sw-language-switch>
```

Comment:
Better use `v-on:change` or `@change`. I think `v-on:on-change` is not valid.

Reply:
I check that but it is copied from production code. Why do you prefer `@` over `v-on:`? I find it `v-on:` easier to understand when you are new to this to see that the `@` is a vue thing.

Reply:
`@` is only syntax sugar. Where do you find `v-on:on-change` in the production code?

Reply:
In MY production code :D I already implemented it for one of our clients.